src/containerbuddy/config.go

```diff
@@ -91,12 +91,12 @@ func loadConfig() *Config {
 		backend.onChangeArgs = strings.Split(backend.OnChangeExec, " ")
 	}
 
-	hostname, _ := os.Hostname()
 	for _, service := range config.Services {
-		service.Id = fmt.Sprintf("%s-%s", service.Name, hostname)
+		hostIP := getIp(service.IsPublic)
+		service.Id = fmt.Sprintf("%s-%s", service.Name, hostIP)
```

Comment:
This changes the format of the `service.Id`, replacing the `hostname` with the `hostIP`, which breaks expectations elsewhere that this is used.